using_openshift/templates.adoc

```diff
@@ -49,20 +49,23 @@ A template describes a set of resources intended to be used together that can be
 <4> A list of labels to apply to resources
 <5> A list of resources to create
 
+====
+
 == Labels
-The list of labels specified in the template will be applied to every resource that is generated from the template. link:../architecture/kubernetes_model.html#label[Labels] can be used to manage and organize generated resources.
+link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template will be applied to every resource that is generated from the template.
 
 == Parameters
-The list of parameters that you want to override are listed in the *parameters* section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+The list of parameters that you can override are listed in the parameters section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
```

Comment:
In "... listed in the parameters section ...", it's referring to the `parameters` section that is part of the template itself, so you could mark it up as:

    `*parameters*`

per [guidelines doc](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) (IMO, fits with "configuration parameter" in this case).